src/bp-core/classes/class-bp-admin-tab.php

```diff
@@ -109,67 +109,84 @@ public function register_admin_script() {
-						'unsuspend_confirm_message' => esc_js( __( 'Please confirm you want to unsuspend this member. Members who are unsuspended will be allowed to login again, and their content will no longer be hidden from other members in your network. Please allow a few minutes for this process to complete.', 'buddyboss' ) ),
+					'repair_forums' => array(
+						'validate_site_id_message' => esc_html__( 'Select site to repair the forums', 'buddyboss' ),
 					),
-					'cover_size_alert'    => array(
-						'profile' => esc_html__( 'Changing the Cover Image Size will reposition all of your members cover images. Are you sure you wish to save these changes?', 'buddyboss' ),
-						'group'   => esc_html__( 'Changing the Cover Image Size will reposition all of your groups cover images. Are you sure you wish to save these changes?', 'buddyboss' ),
-					),
-					'avatar_settings'     => array(
-						'wordpress_show_avatar'    => bp_get_option( 'show_avatars' ),
-						'wordpress_avatar_default' => bp_get_option( 'avatar_default', 'mystery' ),
-						'wordpress_avatar_types'   => array( 'mystery', 'blank', 'gravatar_default', 'identicon', 'wavatar', 'monsterid', 'retro' ),
-					),
-					'profile_group_cover' => array(
-						'select_file'       => esc_js( esc_html__( 'No file was uploaded.', 'buddyboss' ) ),
-						'file_upload_error' => esc_js( esc_html__( 'There was a problem uploading the cover photo.', 'buddyboss' ) ),
-						'feedback_messages' => array(
-							0 => sprintf(
-								/* translators: 1. Cover image width. 2. Cover image height. */
-								esc_html__( 'Cover photo was uploaded successfully. For best results, upload an image that is %1$spx by %2$spx or larger.', 'buddyboss' ),
-								(int) $cover_dimensions['width'],
-								(int) $cover_dimensions['height']
-							),
-							1 => esc_html__( 'Cover photo was uploaded successfully.', 'buddyboss' ),
-							2 => esc_html__( 'There was a problem deleting cover photo. Please try again.', 'buddyboss' ),
-							3 => esc_html__( 'Cover photo was deleted successfully.', 'buddyboss' ),
-						),
-						'upload'            => array(
-							'nonce'           => wp_create_nonce( 'bp-uploader' ),
-							'action'          => 'bp_cover_image_upload',
-							'object'          => ( 'bp-xprofile' === bp_core_get_admin_active_tab() ) ? 'user' : 'group',
-							'item_id'         => 0,
-							'item_type'       => 'default',
-							'has_cover_image' => false,
-						),
-						'remove'            => array(
-							'nonce'  => wp_create_nonce( 'bp_delete_cover_image' ),
-							'action' => 'bp_cover_image_delete',
-							'json'   => true,
+				),
+				'moderation'          => array(
+					'suspend_confirm_message'   => esc_js( __( 'Please confirm you want to suspend this member. Members who are suspended will be logged out and not allowed to login again. Their content will be hidden from all members in your network. Please allow a few minutes for this process to complete.', 'buddyboss' ) ),
```

Comment:
```suggestion
					'suspend_confirm_message'   => esc_js( esc_html__( 'Please confirm you want to suspend this member. Members who are suspended will be logged out and not allowed to login again. Their content will be hidden from all members in your network. Please allow a few minutes for this process to complete.', 'buddyboss' ) ),
```

Reply:
JI - This was not related to this PR but I have done this.